pulser/tests/test_simulation.py

```diff
@@ -659,8 +660,8 @@ def test_get_xy_hamiltonian():
         )
         < 1e-10
     )
-    assert simple_ham[0, 1] == 0.5 * amp
-    assert simple_ham[3, 3] == -2 * detun
+    assert np.isclose(simple_ham[0, 1], 0.5 * amp)
+    assert np.isclose(simple_ham[3, 3], -2 * detun)
```

Comment:
Do you know why it's no longer strictly equal?

Reply:
`simple_ham` was returning a complex number (with zero imaginary part)